src/language/rule/abstractRule.ts

```diff
@@ -34,18 +34,30 @@ export abstract class AbstractRule implements IRule {
             return ruleConfigValue[0];
         }
 
+        if (ruleConfigValue.level !== "off" && ruleConfigValue.level !== "none") {
+            return true;
+        }
+
         return false;
     }
 
     constructor(ruleName: string, private value: any, private disabledIntervals: IDisabledInterval[]) {
         let ruleArguments: any[] = [];
+        let ruleLevel = RuleLevel.ERROR;
```

Comment:
What are you thinking here? Allow the config file to specify a default level for all? Sounds doable.